.gitignore

```diff
@@ -5,6 +5,11 @@ DerivedData
 *.pbxuser
 xcuserdata
 
+# SwiftPackageManager
+Packages
+.build/
+*.xcodeproj
```

Comment:
If this is ignored, then how will people develop on the app? Basically, this deletes this file from the Repo, so now there's nothing to open. Maybe I'm confused by how the development workflow works with SPM?

Reply:
With SPM we don't need a xcodeproj, it is defined by the `Package.swift` file + `.swiftpm`. To load the project just double click `Package.swift` and will open the project.